docs/api.md

```diff
@@ -349,3 +349,11 @@ directory contains collections, events, and index files. A trial checks to see i
 specified in the smdebug hook has been completed. This environment variable
 specifies the maximum number of incomplete steps that the trial will wait for before marking
 half of them as complete. Default: 1000
+
```

Comment:
I feel like all these env variables should have a SMD_ prefix

Reply:
This is technically a backwards incompatible change, but I think we can make an exception here as these were `undocumented` before. But just to be sure, can you add a note at the top saying these are applicable for versions > 0.4.14